public/tenants/planet/manifest.json

```diff
@@ -0,0 +1,52 @@
+{
+    "name": "Plant for the Planet",
+    "short_name": "Plant",
+    "start_url": "/?utm_source=pwa",
+    "description": "Plant trees around the world - Plant-for-the-Planet",
+    "display": "fullscreen",
+    "background_color": "#fff",
+    "theme_color": "#68b030",
+    "lang": "en",
+    "orientation": "portrait",
+    "icons": [
```

Comment:
The transparent icon version does not work well for iPhone and Android devices:

![IMG_9238](https://user-images.githubusercontent.com/1532418/111911057-22caa280-8a64-11eb-894e-3a88113c5744.PNG)![IMG_9240](https://user-images.githubusercontent.com/1532418/111911061-25c59300-8a64-11eb-96c0-305c4123b7be.PNG)

Reply:
I tried to add an additional icons set without transparency for the web manifest, but unfortunately iOS seems to prefer taking the app icons from https://github.com/Plant-for-the-Planet-org/planet-webapp/blob/1a4308e94e6438a6c621655c9443ab57adb9fa42/src/features/common/Layout/Header/index.tsx#L51-L54.

Reply:
using none transparent icons again